lib/gltf.js

```diff
@@ -154,185 +43,301 @@ function createGltf(data, inputPath, modelName) {
     };
 
     gltf.asset = {
-        "generator": "OBJ2GLTF",
-        "premultipliedAlpha": true,
-        "profile": {
-            "api": "WebGL",
-            "version": "1.0"
+        generator : 'obj2gltf',
+        profile : {
+            api : 'WebGL',
+            version : '1.0'
         },
-        "version": 1
+        version: '1.0'
     };
 
     gltf.scenes[sceneId] = {
-        nodes : [nodeId]
+        nodes : []
     };
 
-    gltf.nodes[nodeId] = {
-        children : [],
-        matrix : [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1],
-        meshes : [meshId],
-        name : modelName
-    };
+    function getImageId(imagePath) {
```

Comment:
This might lead to unintended/hard to debug problems for a model with, say, a map in jpg and another map in png named the same thing, like `model.jpg` and `model.png`. It's lousy input, but should we worry about it here?